frontend/src/libs/components/header/header.tsx

```diff
@@ -0,0 +1,35 @@
+import styles from './header.module.scss';
+
+type Properties = {
+  caption: string;
+  username: string;
+  image: string;
```

Comment:
This should came from `user` prop from store

Reply:
I can't reach this user data fields from store right now, can I?

Reply:
All user data (except user image, _image can be a placeholder for not_) is currently in sync - so you can